src/styles.css

```
@@ -407,33 +407,82 @@ color: #FFFF00;
   background: rgba(36, 51, 66, 0.4);
 }
 
-/**/
+/*Pop up*/
+
+.mapboxgl-popup.Male{
+  width: 100px;
+  height: 100px;
+}
+
+
+.mapboxgl-popup.Female{
+  width: 400px;
+}
 
 .mapboxgl-popup-content{
   /* background-color: #0e9686f2; */
   color: white;
   font-weight: bold;
-  max-width: 150px;
   border: 2px solid white;
 }
+
+.mapboxgl-popup.Male .mapboxgl-popup-content{
+  background-color: var(--male-color);
+}
+
+.mapboxgl-popup.Female .mapboxgl-popup-content{
+  background-color: var(--female-color);
+}
 .mapboxgl-popup-close-button{color: white}
+
+.popup-male{
+  font-size: 13px;
+  margin-top: 5px;
+  text-align: center;
+}
+
+.popup-male p{
+  margin: 0;
+}
+
+.popup-female p{
+  margin: 0 0 8px 0;
+}
+
 .popup-female{
-  min-width: 80px;
   margin-top: 5px;
   text-align: center;
   font-size: 14px;
   color: var(--popup-female-text);
 }
 
-.popup-female img{
-  width: 30px;
+
+.popup-female .picture img{
+  width: 100%;
```

I suggest here the use of object-fit and the definition a heigth, even if then image is not fully visualized. This way we keep the ratio across the diferrent popups. 

```suggestion
  width: 100%;
  height: 250px;
  object-fit: cover;
```